### DISCUSSION TITLE
挂载的阿里云盘双击打不开

### DISCUSSION THREAD

**Discussion by YUNNANZHOU** [AlistGo/alist#6120] [Q&A]
![image](https://github.com/alist-org/alist/assets/73827885/1d44ee8d-9f44-4c1c-9094-d256bd529257)

**Comment by YUNNANZHOU** [comment]
自带的管理双击打不开啊，一直停留在这个界面。

**Comment by AspirantH** [comment]
3.31版本一旦打开复选框就不能点击了

**Reply by YUNNANZHOU** [reply]
原来如此，感谢兄弟！我关闭复选，就能点进去了。

**Comment by zhsxh** [comment]
我也是存在这个问题，网盘名字前面有复选框，导致无法进入网盘

**Reply by YUNNANZHOU** [reply]
原来如此，感谢兄弟！我关闭复选，就能点进去了。
![image](https://github.com/alist-org/alist/assets/73827885/70c850be-fdeb-47fc-86c6-6595b267c4e8)

**Reply by zhsxh** [reply]
怎么关闭复选啊？

**Comment by zhsxh** [comment]
怎么关闭复选啊？我还没学会，教教我呗。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月1日(星期五) 中午11:21
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] 挂载的阿里云盘双击打不开 (Discussion #6120)





 
原来如此，感谢兄弟！我关闭复选，就能点进去了。
 image.png (view on web)

**Reply by YUNNANZHOU** [reply]
右下角

**Comment by zhsxh** [comment]
看到了，非常感谢。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月1日(星期五) 中午12:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] 挂载的阿里云盘双击打不开 (Discussion #6120)





 
右下角
